### ISSUE TITLE
Unable to put description below title. 

### ISSUE THREAD

**Issue by arvsr1988** [AppIntro/AppIntro#349]
**4.1.0*:

**Issue details / Repro steps / Use case background**: 
I want to put the description below the title as shown in Layout 2 of the Readme.MD. However, it looks like the description will always come below the image (unless I create my own fragment). On looking into the code it looks like AppIntro and AppIntro2 use the same layout. Maybe that is why. 

**Your Code**: 
```
public class IntroActivity extends AppIntro2 {
        addSlide(AppIntro2Fragment.newInstance("Test title 1", "", R.drawable.circles, getResources().getColor(R.color.colorPrimaryDark)));
        addSlide(AppIntro2Fragment.newInstance("Test title 2", "", R.drawable.circles, getResources().getColor(R.color.colorPrimaryDark)));
}
```

**Comment by avluis** [comment]
OK... Thanks for letting us know.
What's keeping you from putting the description below the title?
Carpal Tunnel? Your Cat? PEBKAC?

If you look closely at AppIntro and AppIntro2, you should be able to see that a tiny difference between the navigation buttons (you may need to grab the sample app and run it a few times until you see it -- you'll 💩 when you see it).

@Andrew-Quebe Is this for real or troll?

**Comment by Andrew-Quebe** [comment]
@avluis I'm not sure about the tiny difference in navigation buttons you're talking about...but I do know that we should add an option for putting text above or below the image. A `textPosition(position)` method might be good.

**Comment by Anutha02** [comment]
@cortinico Add this to the Triage, I'm working on it

**Comment by Anutha02** [comment]
@cortinico Fixed by #661

**Comment by cortinico** [comment]
Closing as too old + can be easily achieved with a custom Layout fragment.
